### ISSUE TITLE
Thoughts on migrating to SvelteKit?

### ISSUE THREAD

**Issue by Brisklemonade** [mskocik/svelty-picker#12]
Have you considered migrating the project to SvelteKit? SvelteKit come with a Package command that would make your job easier. On top of that, the docs can be built on SvelteKit + you can enable typescript support. If you're cool with doing that, I would love to help contribute.

**Comment by mskocik** [comment]
Yes, it would be great to have it. But I haven't found time for it yet. Feel free to prepare it. Will hapilly accept that PR

**Comment by Crypt0Jesus** [comment]
Any chance of release soon?  This is the picker I really want to use.

**Comment by mskocik** [comment]
SvelteKit version (separate branch) is broken and needs fixing. You can help. I have no time at the moment for that.

**Comment by Brisklemonade** [comment]
> Any chance of release soon?  This is the picker I really want to use.

SvelteUI will have a date picker component soon. You should be on the look out for that!

**Comment by mskocik** [comment]
@Crypt0Jesus I've already released SvelteKit version with many updates. Unfortunately redesigned docs are not one of them 😅 
Hopefully soon will get to that, if no one will do it sooner 😇
